keptn-gateway/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+
+	"fmt"
+	"log"
+	"net/http"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	metrics_v1 "github.com/keptn/lifecycle-toolkit/metrics-operator/api/v1"
+)
+
+var clientset *kubernetes.Clientset
+
+func handlerMetricsCount(w http.ResponseWriter, r *http.Request) {
+	log.Println("http call to /pods")
+
+	d, err := clientset.RESTClient().
+		Get().
+		AbsPath("/apis/metrics.keptn.sh/v1/").
+		Resource("keptnmetrics").
+		Namespace("default").
+		DoRaw(context.TODO())
+
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+    _, err = w.Write([]byte(err.Error()))
+    if err != nil {
+      fmt.Println(err.Error())
+    }
+	}
+
+	l := metrics_v1.KeptnMetricList{}
+
+	err = json.Unmarshal([]byte(d), &l)
+	if err != nil {
+		fmt.Println(err.Error())
+	}
+
+	w.WriteHeader(http.StatusOK)
+	returnval := fmt.Sprintf("%#v", len(l.Items))
+
+	_, err = w.Write([]byte(returnval))
+	if err != nil {
+		fmt.Println(err.Error())
+	}
+}
+
+func main() {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		panic(err.Error())
+	}
+	// creates the clientset
+	clientset, err = kubernetes.NewForConfig(config)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	http.Handle("/metricscount", http.HandlerFunc(handlerMetricsCount))
```

Comment:
how many endpoints do we envision? If it is more than one or two, we should maybe look into Gin, FastHTTP or other frameworks 🤔

Reply:
2 at the moment